Author: P Stahlman <https://launchpad.net/~pstahlman>
Description: Increase buffer to cater for maximum file name length (LP: #179684).
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/unace-nonfree/+bug/179684
Origin: https://launchpadlibrarian.net/35753473/acefuncs.c.patch

--- unace-nonfree-2.5.orig/source/apps/exe/acefuncs/acefuncs.c
+++ unace-nonfree-2.5/source/apps/exe/acefuncs/acefuncs.c
@@ -27,7 +27,7 @@
 
 void    APPS_EXE_ACEFUNCS_List(BOOL Verbose)
 {
-CHAR      ShortStr[80],
+CHAR      ShortStr[BASE_LFN_MAXLEN],
           SizeStr1[80],
           SizeStr2[80],
           OutputStr[160];
